Assets/MixedRealityToolkit/_Core/Utilities/MotionController/WindowsMixedRealityControllerInfo.cs

```diff
@@ -7,10 +7,10 @@
 namespace Microsoft.MixedReality.Toolkit.Internal.Utilities
 {
     /// <summary>
-    /// This script keeps track of the GameObjects for each button on the controller.
+    /// This script keeps track of the GameObjects representations for each button on the Windows Mixed Reality Controllers.
     /// It also keeps track of the animation Transforms in order to properly animate according to user input.
     /// </summary>
-    public class MotionControllerInfo
+    public class WindowsMixedRealityControllerInfo
```

Comment:
why did this shift from a generic representation to be specific to Windows Mixed Reality controllers?

Reply:
This is still the old WMRControllerInfo stuff @davidkline-ms , so it will eventually itself be refactored out most likely.  baby steps.
MotionControllerInfo it'self shouldn't exist anymore.

Reply:
Because it wasn't a generic representation.